packages/contracts/contracts/universal/IMultiSigWallet.sol

```diff
@@ -0,0 +1,195 @@
+     *
+     * @param _owner Address of owner.
+     */
+    function removeOwner(address _owner) external;
+
+    /**
+     * @notice Allows to replace an owner with a new owner. Transaction has to be sent by wallet.
+     *
+     * @param _owner    Address of owner to be replaced.
+     * @param _newOwner Address of new owner.
+     */
+    function replaceOwner(address _owner, address _newOwner) external;
+
+    /**
+     * @notice Allows to change the number of required confirmations.
+     *         Transaction has to be sent by wallet.
+     *
+     * @param _required Number of required confirmations.
+     */
+    function changeRequirement(uint256 _required) external;
+
+    /**
+     * @notice Allows an owner to submit and confirm a transaction.
+     *
+     * @param _destination Transaction target address.
+     * @param _value       Transaction ether value.
+     * @param _data        Transaction data payload.
+     *
+     * @return Returns transaction ID.
+     */
+    function submitTransaction(
+        address _destination,
+        uint256 _value,
+        bytes memory _data
+    ) external returns (uint256);
+
+    /**
+     * @notice Allows an owner to confirm a transaction.
+     *
+     * @param _transactionId Transaction ID.
+     */
+    function confirmTransaction(uint256 _transactionId) external;
+
+    /**
+     * @notice Allows an owner to revoke a confirmation for a transaction.
+     *
+     * @param _transactionId Transaction ID.
+     */
+    function revokeConfirmation(uint256 _transactionId) external;
+
+    /**
+     * @notice Allows anyone to execute a confirmed transaction.
+     *
+     * @param _transactionId Transaction ID.
+     */
+    function executeTransaction(uint256 _transactionId) external;
+
+    /**
+     * @notice Returns the confirmation status of a transaction.
+     *
+     * @param _transactionId Transaction ID.
+     *
+     * @return Confirmation status.
+     */
+    function isConfirmed(uint256 _transactionId) external view returns (bool);
+
+    /**
+     * @notice Web3 call functions. Returns number of confirmations of a transaction.
+     *
+     * @param _transactionId Transaction ID.
+     *
+     * @return Number of confirmations.
+     */
+    function getConfirmationCount(uint256 _transactionId) external view returns (uint256);
+
+    /**
+     * @notice Returns total number of transactions after filers are applied.
+     *
+     * @param _pending  Whether include pending transactions.
+     * @param _executed Whether include executed transactions.
+     *
+     * @return Total number of transactions after filters are applied.
+     */
+    function getTransactionCount(bool _pending, bool _executed) external view returns (uint256);
+
+    /**
+     * @notice Returns list of owners.
+     *
+     * @return addresses The list of owner addresses.
+     */
+    function getOwners() external view returns (address[] memory addresses);
```

Comment:
Could you remove return variable names of these 3 functions from here, to match the actual implement in `MultiSigWallet.sol`?

Reply:
Applied.